.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: true
```

Comment:
Please explain.

Reply:
This just give the user an option to open a blank templateless issue and post it. He'll just have an option to neither choose bug report nor feature request.

Reply:
ok, lets keep it for now, we will see if users misuse it .